docs/src/main/paradox/bootstrap.md

```diff
@@ -62,6 +64,14 @@ akka.discovery.method = akka-dns
 Be sure to read about the @ref[alternative discovery methods](discovery.md) that Akka provides, and you
 can pick the one most fitting to your cluster manager or runtime -- such as Kubernetes, Mesos or other cloud or tool specific methods.
 
+The http routes used for bootstrap are automatically added to Akka management. Your application will need to start Akka management:
+
+@@snip [akka-cluster.yml](../../../../bootstrap-joining-demo/local/src/main/scala/akka/cluster/bootstrap/Main.scala) { #akka-management }
+
+Then to start the boot strap process:
+
+@@snip [akka-cluster.yml](../../../../bootstrap-joining-demo/local/src/main/scala/akka/cluster/bootstrap/Main.scala) { #bootstrap }
+
```

Comment:
Thanks for adding this! I think a few people have tripped over this already.

Reply:
Hmm, quite a waste to close this PR without taking these additions... I'll revive them